staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1/types.go

```diff
@@ -248,6 +248,8 @@ const (
 
 // DataVolumeStatus contains the current status of the DataVolume
 type DataVolumeStatus struct {
+	// ClaimName is the name of the underlying PVC used by the DataVolume.
+	ClaimName string `json:"claimName,omitempty"`
 	//Phase is the current phase of the data volume
```

Comment:
Why put it in status and not in spec? that way one can actually choose a non-default name for the created PVC.
Currently the field is static once set by `reconcileDataVolumeStatus()`, and that's a bit boring for status imho.

Reply:
Things will really start breaking if we allow that, right now a lot of things implicitly assume pvc name == dv name :-)

Reply:
and now, in places that assume pvc name == dv name, we can start using the status.claimName, and only in the future we might have the possibility to introduce the spec.pvcName. 

Anyway changing `spec.pvcName` might only be possible before the actual action(upload/import/clone etc...) starts. But that is a future ;)